core/src/main/scala/org/apache/spark/metrics/MetricsSystem.scala

```diff
@@ -201,6 +203,12 @@ private[spark] class MetricsSystem private (
                 classOf[Properties], classOf[MetricRegistry], classOf[SecurityManager])
               .newInstance(kv._2, registry, securityMgr)
             metricsServlet = Some(servlet)
+          } else if (kv._1 == "prometheusServlet") {
```

Comment:
If you want you can make `kv` into `case (key, value)` here for clarity, but not necessary